pilot/pkg/networking/core/v1alpha3/route/route_test.go

```diff
@@ -71,9 +72,50 @@ func TestBuildHTTPRoutes(t *testing.T) {
 	t.Run("for virtual service", func(t *testing.T) {
 		g := gomega.NewGomegaWithT(t)
 
+		os.Setenv("ISTIO_DEFAULT_REQUEST_TIMEOUT_MS", "0")
+		defer os.Unsetenv("ISTIO_DEFAULT_REQUEST_TIMEOUT_MS")
+
+		routes, err := route.BuildHTTPRoutesForVirtualService(node, nil, virtualServicePlain, serviceRegistry, 8080, gatewayNames)
+		g.Expect(err).NotTo(gomega.HaveOccurred())
+		g.Expect(len(routes)).To(gomega.Equal(1))
+		// Validate that when timeout is not specified, we disable it based on default value of flag.
+		g.Expect(routes[0].GetRoute().Timeout.Seconds).To(gomega.Equal(int64(0)))
+		g.Expect(routes[0].GetRoute().MaxGrpcTimeout.Seconds).To(gomega.Equal(int64(0)))
+	})
+
+	t.Run("for virtual service with changed default timeout", func(t *testing.T) {
+		g := gomega.NewGomegaWithT(t)
+
+		os.Setenv("ISTIO_DEFAULT_REQUEST_TIMEOUT_MS", "1000ms")
```

Comment:
Shouldn't use 1000?

Reply:
For duration type of var, I think it needs to specify time unit - otherwise it throws error

Reply:
After reading the lib, you are right. It's the name that just makes me confused. 
How about `ISTIO_DEFAULT_REQUEST_TIMEOUT`? As it can be set as `1s` or even `1m`

Reply:
That makes sense. I will change